### DISCUSSION TITLE
Checkboxes not working properly with dark reader plugin

### DISCUSSION THREAD

**Discussion by jquinterog** [twilio-labs/paste#2165] [Q&A] [answered]
Checkboxes have a weird behaviour while using dark mode with the plugin "dark reader" in chrome, I'm not sure if it's something from paste or the dark reader, can you check it?
This is the plugin https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh
And the paste checkboxes https://paste.twilio.design/components/checkbox/#unchecked-checked-and-indeterminate

**Comment by shleewhite** [comment, marked as answer]
Hi @jquinterog! Thank you for bringing this up, it looks like currently the check icon is hidden by making it the same color as the background. Since this is not the ideal way to hide content, I have opened a PR to fix this: https://github.com/twilio-labs/paste/pull/2167

I will let you know when the fix is merged!

**Reply by shleewhite** [reply]
Hi @jquinterog, the fix just merged! The new minor version of Checkbox should be available soon, so if you update the packages you will see the checkboxes correctly.
